v3/src/components/axis/axis-utils.ts

```diff
@@ -0,0 +1,69 @@
+import {AxisPlace} from "./axis-types"
+
+
+export const getCategoricalLabelPlacement = (
+  axisPlace: AxisPlace, centerCategoryLabels: boolean, collision: boolean, bandWidth: number, textHeight: number) => {
```

Comment:
With more than about three arguments I generally switch to passing an object rather than a list of comma-separated arguments, but it's a judgment call.

Reply:
Thanks for pushing me on this. I was being lazy!